src/main/scala/group/scala/karazin/mongo4s/model.scala

```diff
@@ -124,12 +124,12 @@ object model:
                                                   filter: Option[Filter] = None,
                                                   sort: Option[Sort] = None,
                                                   projection: Option[Projection] = None,
-                                                  hint: Option[Either[String, Hint]] = None,
+                                                  hint: Option[Hint] = None,
                                                   skip: Option[Int] = None,
                                                   limit: Option[Int] = None,
                                                   batchSize: Option[Int] = None,
                                                   singleBatch: Option[Boolean] = None,
-                                                  comment: Option[Any] = None,
+                                                  comment: Option[JsonObject] = None,
```

Comment:
from the docs:
`A comment can be any valid BSON type (string, integer, object, array, etc).`

Reply:
Good point. I missed the point that JsonObject does not suit all of these types.